command/pr.go

```diff
@@ -97,8 +97,8 @@ func prStatus(cmd *cobra.Command, args []string) error {
 	fmt.Fprintln(out, "")
 
 	printHeader(out, "Current branch")
-	if prPayload.CurrentPR != nil {
-		printPrs(out, 0, *prPayload.CurrentPR)
+	if prPayload.CurrentPRs != nil {
+		printPrs(out, 0, prPayload.CurrentPRs...)
```

Comment:
Hi, little late for a review, but I'm puzzled about the decision why would we ever _show_ more than one result for PRs that belong to the current branch? A single branch can only ever have maximum of 1 open pull request attached to it. There may be more than 1 cloned/merged PRs attached to a single branch name, but I'm not sure why we would want to list them in addition to the open one? I thought that this PR would be just making sure that the *latest* PR appears in the "Current branch" section even if it's closed or merged.

Reply:
@mislav 
Ahhh, I see. I must have misunderstood what you meant at https://github.com/cli/cli/pull/530#discussion_r383444563. 🙇 Let me fix it.